teamchains/hidden_missing_reverse_sig.json

```diff
@@ -210,12 +222,12 @@
             ],
             "hidden": [
                 {
-                    "i": "h6FigaFrkYehYQGhY4KhaMQgc6m8wRG48BO8aYiASY34Y1RAMCpqprwu4eIh0QpcKQqhdgGhZcQjASFD8wqUnrwojc9r0u7o/SqeVkG3GP9h2GDoK6R8J1bKWQqhZwOhcsChc8QjASBEenIc53ynalOzD9lPOS1tstLBA9gYyo3KqBIJVdmV9AqhdAChY85ZcOGRoWXEEAAAAAAAAAAAAAAAAAAAAAChbYOhY85ZcOGRoWjEIM3Nzc3Nzc3Nzc3Nzc3Nzc3Nzc3Nzc3Nzc3Nzc3NzcmRoXPNH0GhcIOhaMQg+0zF4G2D1n0yYQhiOh0bpOPwstNNR32kmcEvOP5BDkuhcwKhdAOhc4OhZQGha8QjASASYM8VuArq8sLDYvDeiLvEAnLVNNF5uUwEV6XlhPzRhgqhdcQQJYUsh9bkf7jX1VQAvpx6GaF0g6FpxBAhIyCbmLFgg8ackRUrhhckoW3CoXDC",
-                    "o": "mAMBwMQgIdP/5dFrEh5NdQu+sJMtlqQX8ZZ1etej04OSdvopGzxREcLA",
-                    "s": "xEAZ0MPL0PCrS007yGyhsZbfptbPSHZs1vlbkgKFDmuFqvyEh6SCJS7b1nh0DAjJJhLsQxdoEKkbTKU7KCE6NkIA"
+                    "i": "h6FigaFrkYehYQGhY4KhaMQgc6m8wRG48BO8aYiASY34Y1RAMCpqprwu4eIh0QpcKQqhdgGhZcQjASFD8wqUnrwojc9r0u7o/SqeVkG3GP9h2GDoK6R8J1bKWQqhZwOhcsChc8QjASBEenIc53ynalOzD9lPOS1tstLBA9gYyo3KqBIJVdmV9AqhdAChY85ZcOGRoWXEEAAAAAAAAAAAAAAAAAAAAAChbYOhY85ZcOGRoWjEIM3Nzc3Nzc3Nzc3Nzc3Nzc3Nzc3Nzc3Nzc3Nzc3NzRmCoXPNH0GhcIOhaMQg+0zF4G2D1n0yYQhiOh0bpOPwstNNR32kmcEvOP5BDkuhcwKhdAOhc4OhZQGha8QjASASYM8VuArq8sLDYvDeiLvEAnLVNNF5uUwEV6XlhPzRhgqhdcQQJYUsh9bkf7jX1VQAvpx6GaF0g6FpxBAhIyCbmLFgg8ackRUrhhckoW3CoXDC",
```

Comment:
did you ever get to the bottom of what changed here?

Reply:
ok, it's the bottom byte on hash_meta, which changes as a function of the chains, i think. so it's ok.